samples/bluetooth/direct_test_mode/src/dtm.c

```diff
@@ -1332,7 +1333,16 @@ static enum dtm_err_code phy_set(uint8_t phy)
 		anomaly_172_strict_mode_set(false);
 		nrfx_timer_disable(&dtm_inst.anomaly_timer);
 		dtm_inst.anomaly_172_wa_enabled = false;
-#endif
+#endif /* NRF52840_XXAA */
+
+#ifdef NRF53_ERRATA_117_ENABLE_WORKAROUND
+		/* Workaround for Errata 117 "RADIO: Changing MODE requires additional
+		 *  configuration" found at the Errata document for your device located at
+		 * https://infocenter.nordicsemi.com/index.jsp
+		 */
+		*((volatile uint32_t *)0x41008588) = *((volatile uint32_t *)0x01FF0080);
+#endif /* NRF53_ERRATA_117_ENABLE_WORKAROUND */
```

Comment:
Consider defining a dedicated function for handling the errata activation (if necessary). The function can be called _errata_117_handle_. This way, we won't have as much boilerplate code and comment descriptions.

Reply:
Good idea. I added a function for handling this errata